examples/misc/test/effective_dart_test.dart

```diff
@@ -13,4 +13,43 @@ void main() {
     expect('abcd'.substring(1, 3), /**/ 'bc');
     // #enddocregion param-range
   });
+
+  test('toList()', () {
+    {
+      var iterable = [1, 2];
+      // #docregion list-from-1
+      var copy1 = iterable.toList();
+      var copy2 = new List.from(iterable);
+      // #enddocregion list-from-1
+      expect(copy1, orderedEquals([1, 2]));
+      expect(copy2, orderedEquals([1, 2]));
+    }
+
+    {
+      // #docregion list-from-2
+      // Creates a List<int>:
+      var iterable = [1, 2, 3];
+
+      // Prints "List<int>":
+      print(iterable.toList().runtimeType);
+
+      // Prints "List", which means List<dynamic>:
+      print(new List.from(iterable).runtimeType);
+      // #enddocregion list-from-2
+
+      // TODO: Enable this check when this test is run in Dart 2 mode.
+      // expect(iterable.toList().runtimeType.toString(), "List<int>");
+      expect(new List.from(iterable).runtimeType.toString(), "List");
```

Comment:
There are slightly easier ways to test snippets w/ prints using the `prints` matcher, but I can address that later. Here is an example, in case you are wondering: https://github.com/dart-lang/site-www/blob/master/examples/misc/test/language_tour/functions_test.dart#L86

Reply:
That would be great, thanks. I'm trying to get through as many changes as I can before I get sucked onto working on other things. :)